docs/usage/getting-started/running.md

```diff
@@ -153,14 +153,18 @@ Regardless of platform, you need to select a user account for `renovate` to assu
 It is recommended to be `@renovate-bot` if you are using a self-hosted server with free choice of usernames.
 It is also recommended that you configure `config.gitAuthor` with the same identity as your Renovate user, e.g. like `"gitAuthor": "Renovate Bot <renovate@whitesourcesoftware.com>"`.
 
-#### GitHub (Enterprise Server)
+#### Docs
 
-First, [create a Personal Access Token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) for the bot account (select "repo" scope).
-Configure it either as `token` in your `config.js` file, or in environment variable `RENOVATE_TOKEN`, or via CLI `--token=`.
+Read the platform-specific docs to learn how to setup authentication on your platform:
 
-For GitHub Enterprise Server set the `endpoint` in your `config.js` to `https://github.enterprise.com/api/v3/`.
+- [Azure DevOps](https://docs.renovatebot.com/modules/platform/azure/)
+- [Bitbucket Cloud](https://docs.renovatebot.com/modules/platform/bitbucket/)
+- [Bitbucket Server](https://docs.renovatebot.com/modules/platform/bitbucket-server/)
+- [Gitea](https://docs.renovatebot.com/modules/platform/gitea/)
+- [GitHub (Enterprise Server)](https://docs.renovatebot.com/modules/platform/github/)
```

Comment:
This platform is either github.com or GHES

Reply:
```suggestion
- [github.com or GHES](https://docs.renovatebot.com/modules/platform/github/)
```